routes/routes.js

```diff
@@ -0,0 +1,95 @@
+const express = require('express');
+const jsonParser = require('body-parser').json();
+const Wizard = require(`${__dirname}/../models/wizards.js`);
+const Film = require('../models/films.js');
+const sendMessage = require('../lib/sendMessage.js');
+
+const Router = module.exports = express.Router();
```

Comment:
probably shouldn't be an uppercase first letter since this is an instance of the Router constructor and not a constructor itself.